site/topics/testing/testing.rst

```diff
@@ -0,0 +1,9 @@
+*****************
+Testing and Hints
+*****************
```

Comment:
Maybe _Testing Your Code and Type Hints_

Title sounds more like hints for the upcoming exam.

Reply:
hahaha. Good point.